### PR TITLE
feat(DataArts): add dataarts data standard resource

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  add dataarts data standard resource
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  add dataarts data standard resource
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=./huaweicloud/services/acceptance/dataarts/ TESTARGS='-run TestAccDataStandard_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dataarts/ -v -run TestAccDataStandard_basic -timeout 360m -parallel 4
=== RUN   TestAccDataStandard_basic
=== PAUSE TestAccDataStandard_basic
=== CONT  TestAccDataStandard_basic
--- PASS: TestAccDataStandard_basic (26.36s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dataarts  26.406s
```

### CODE REVIEW

File: huaweicloud/services/dataarts/resource_huaweicloud_dataarts_studio_data_standard.go [hunk 1]
```diff
@@ -0,0 +1,582 @@
...
+
+func dataStandardValueSchema() *schema.Resource {
+	sc := schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"fd_name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the name of the data standard attribute.`,
+			},
```
Comment (Lance52259) [inline huaweicloud/services/dataarts/resource_huaweicloud_dataarts_studio_data_standard.go]: The behavior of this parameter is required in the API request body
The document ditto.

File: huaweicloud/services/acceptance/dataarts/resource_huaweicloud_dataarts_studio_data_standard_test.go [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+  values {
+    fd_name  = "nameCh"
+    fd_value = "%[3]s"
+  }
+
+  values {
+    fd_name  = "nameEn"
+    fd_value = "%[3]s"
+  }
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/dataarts/resource_huaweicloud_dataarts_studio_data_standard_test.go]: can we testing the empty value?

File: huaweicloud/services/dataarts/resource_huaweicloud_dataarts_studio_data_standard.go [hunk 3]
```diff
@@ -0,0 +1,582 @@
...
+			"biz_type": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the type of the new biz.`,
+			},
+			"biz_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the ID of data standard.`,
+			},
```
Comment (Lance52259) [inline huaweicloud/services/dataarts/resource_huaweicloud_dataarts_studio_data_standard.go]: According the description of the parameter `biz_type`, does the descriptions of the parameter `id` and `biz_id` are comfused?

File: huaweicloud/services/dataarts/resource_huaweicloud_dataarts_studio_data_standard.go [hunk 4]
```diff
@@ -0,0 +1,582 @@
...
+		"ids": []string{d.Id()},
+	}
+	return bodyParams
+}
+
+// resourceDataStandardImportState use to import an id with format <workspace_id>/<id>
+func resourceDataStandardImportState(_ context.Context, d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData, error) {
+	if !strings.Contains(d.Id(), "/") {
+		return []*schema.ResourceData{d}, nil
+	}
```
Comment (Lance52259) [inline huaweicloud/services/dataarts/resource_huaweicloud_dataarts_studio_data_standard.go]: There is unnecessary to check whether id string contains character `/`

File: huaweicloud/services/dataarts/resource_huaweicloud_dataarts_studio_data_standard.go (hunk 1 above)
Comment (houpeng80) [reply huaweicloud/services/dataarts/resource_huaweicloud_dataarts_studio_data_standard.go]: done

File: huaweicloud/services/acceptance/dataarts/resource_huaweicloud_dataarts_studio_data_standard_test.go (hunk 2 above)
Comment (houpeng80) [reply huaweicloud/services/acceptance/dataarts/resource_huaweicloud_dataarts_studio_data_standard_test.go]: done

File: huaweicloud/services/dataarts/resource_huaweicloud_dataarts_studio_data_standard.go (hunk 4 above)
Comment (houpeng80) [reply huaweicloud/services/dataarts/resource_huaweicloud_dataarts_studio_data_standard.go]: done

**Comment by houpeng80** [comment]
make testacc TEST=./huaweicloud/services/acceptance/dataarts/ TESTARGS='-run TestAccDataStandard_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dataarts/ -v -run TestAccDataStandard_basic -timeout 360m -parallel 4
=== RUN   TestAccDataStandard_basic
=== PAUSE TestAccDataStandard_basic
=== CONT  TestAccDataStandard_basic
--- PASS: TestAccDataStandard_basic (21.99s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dataarts  22.041s

**Pull request merged**